src/core/swap/swap-api.js

```diff
@@ -25,13 +25,12 @@ export async function fetchSwapQuote(
   const { preferPluginId, disabled = {}, promoCodes = {} } = opts
   const { log } = ai.props
 
-  log.warn(
-    `EdgeSwapRequest fromCurrencyCode ${request.fromCurrencyCode} toCurrencyCode ${request.toCurrencyCode} nativeAmount ${request.nativeAmount} quoteFor ${request.quoteFor}`
-  )
   const account = ai.props.state.accounts[accountId]
   const { swapSettings, userSettings } = account
   const swapPlugins = ai.props.state.plugins.swap
 
+  console.warn('Requesting swap quotes for: ', request)
```

Comment:
???

Reply:
OOops, should be `log.warn`. Told ya it was untested.